lib/autoparts/packages/rust.rb

```diff
@@ -9,12 +9,12 @@ class Rust < Package
       description 'Rust: A safe, concurrent, practical language'
       category Category::PROGRAMMING_LANGUAGES
 
-      source_url 'http://static.rust-lang.org/dist/rust-0.12.0-x86_64-unknown-linux-gnu.tar.gz'
-      source_sha1 'af9011e3cdff0f4de81d8d4eefe85d518731b2b1'
+      source_url 'https://static.rust-lang.org/dist/rust-1.0.0-alpha-x86_64-unknown-linux-gnu.tar.gz'
```

Comment:
It's the first milestone where the language features are stabilized. Moving forward releases will be focused on usability improvements (library api's etc).